proprietary_secure_channel/README.md

```diff
@@ -0,0 +1,245 @@
+# The Remote Attestation Sample
+
+This sample demonstrates how to do remote attestation between two enclaves.
+
+It has the following properties:
+
+- Written in C++
+- Demonstrates an implementation of remote attestation
+- Use of mbedTLS within the enclave
+- Use Asymmetric / Public-Key Encryption to establish secure communications between two attesting enclaves
```

Comment:
Do you want to mention symmetric session key used in the process?

Reply:
Yes, I need to mention this. Making a few more additions to README.md including adding the new flow.

Reply:
Done